lir/config/experiment_strategies.py

```diff
@@ -77,31 +74,47 @@ def primary_metric(self) -> Callable:
         metric_name = pop_field(self._config, 'primary_metric')
         return parse_metric(metric_name, self._output_dir, self._config.context)
 
-    def aggregations(self) -> Sequence[Aggregation]:
-        aggregation = pop_field(self._config, 'aggregation', required=False)
-        if not aggregation:
+    def output_list(self) -> Sequence[Aggregation]:
+        config: ContextAwareDict = pop_field(self._config, 'output', required=False)
+        if not config:
             return []
 
         results: list[Aggregation] = []
+        for item in config:
+            # Normalise configuration into (class_name, args)
+            if isinstance(item, str):
+                class_name, args = item, ContextAwareDict(config.context)
```

Comment:
```suggestion
        for i, item in enumerate(config):
            # Normalise configuration into (class_name, args)
            if isinstance(item, str):
                class_name, args = item, ContextAwareDict(config.context + [str(i)])
```

In het geval van een error wijst dit de gebruiker preciezer naar de oorzaak.